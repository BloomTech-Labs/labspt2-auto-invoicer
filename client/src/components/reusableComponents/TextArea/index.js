import React from "react";

const TextArea = props => {
  return (
    <div>
      <label className="form-label">{props.title}</label>
      <textarea
        className="form-input"
        name={props.name}
        rows={props.rows}
        value={props.content}
        onChange={props.controlFunc}
        placeholder={props.placeholder}
      />
    </div>
  );
};

export default TextArea;
