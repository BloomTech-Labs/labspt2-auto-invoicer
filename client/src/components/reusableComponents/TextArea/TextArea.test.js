import React from "react";
import { render } from "react-testing-library";
import "jest-dom/extend-expect";
import TextArea from "../TextArea";

describe("<TextArea />", () => {
  it("renders TextArea", () => {
    const { asFragment } = render(<TextArea />);
    expect(asFragment()).toMatchSnapshot();
  });
});
