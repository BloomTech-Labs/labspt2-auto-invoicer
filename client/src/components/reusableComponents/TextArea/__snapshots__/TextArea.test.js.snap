// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextArea /> renders TextArea 1`] = `
<DocumentFragment>
  <div>
    <label
      class="form-label"
    />
    <textarea
      class="form-input"
    />
  </div>
</DocumentFragment>
`;
